scipy/stats/contingency.py

```diff
@@ -1,13 +1,29 @@
-"""Some functions for working with contingency tables (i.e. cross tabulations).
+"""
+Contingency table functions (:mod:`scipy.stats.contingency`)
+============================================================
+
+Functions for creating and analyzing contingency tables.
+
+.. currentmodule:: scipy.stats.contingency
+
+.. autosummary::
+   :toctree: generated/
+
+   chi2_contingency
+   expected_freq
+   margins
+   relative_risk
```

Comment:
Why no fisher_exact?

Reply:
Never mind, I see that `fisher_exact` is not actually in the namespace. Should it be?